pysubyt/j2/generator.py

```diff
@@ -30,6 +30,9 @@ def __init__(self, templates_folder: str):
             ),
         )
         self._templates_env.globals = Functions.all()
+        # Since  the change ttl -> xsd, and removal of ttl to filters
+        # But this seems a bit redundact.
+        self._templates_env.globals.update(Filters.all())
```

Comment:
better handle this in pyrdfj2? @marc-portier

Reply:
https://github.com/vliz-be-opsci/pyrdfj2/issues/9

Reply:
joining with Bram on this one